MatiasDure/Project_IMTS
# Bee Transition Out Plot

**Description**
* When user is far enough from the tracked images, the plots should deactivate and the bee should start following the camera again.

**Screenshot**
![BeeTransitionOutPlot](https://github.com/user-attachments/assets/f571f66d-5a58-459f-aded-0e484e880ef8)

**How To Test**
1. checkout the `bee-transition-out-plot`
2. Switch to the `PlaytestPreparation` scene
3. Run the scene
4. Scan the image
5. Bee should go inside the plot 
6. Go back enough (10 unity units)
7. Plot should deactivate and bee should start following the camera again
8. Get close the image without facing the image (plot should not activate)
9. Face the image and plot should re activate and trigger the bee to go through the portal again
10. Try out things to break it
